common/static/js/capa/schematicinput.js

```diff
@@ -1,8 +1,3 @@
-var schematic_height = 220;
-var schematic_width = 400;
-var styling_height_delta = 2; //How many pixels are added to the height of the box because of styling (like a shadow)
-var styling_width_delta = 2;
```

Comment:
These (global!) variables are are only used in the removed code from `common/static/js/vendor/CodeMirror/addons/edx_markdown.js`.